### ISSUE TITLE
visual indicator of session status

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#114]
an icon or word or possibly both/either should show up somewhere on the screen whenever the client is either hosting (even if no one is connected) or visiting

**Comment by HeladoDeBrownie** [comment]
does this mean it should be a *session* view after all instead of a scene view
